### DISCUSSION TITLE
为啥火弦月的8gt刷沙机用不了

### DISCUSSION THREAD

**Discussion by TSCkeineiwi** [LeavesMC/Leaves#321] [Q&A]
末地门处使用刷沙机沙子两次后就会变成掉落物

**Comment by Bluemangoo** [comment]
什么版本？

**Comment by TSCkeineiwi** [comment]
1.20.6






------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2024年8月23日(星期五) 晚上7:35
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [LeavesMC/Leaves] 为啥火弦月的8gt刷沙机用不了 (Discussion #321)














 
什么版本？

**Reply by LittleChest** [reply]
> 1.20.6
> […](#)
> ------------------&nbsp;原始邮件&nbsp;------------------ 发件人: ***@***.***&gt;; 发送时间: 2024年8月23日(星期五) 晚上7:35 收件人: ***@***.***&gt;; 抄送: ***@***.***&gt;; ***@***.***&gt;; 主题: Re: [LeavesMC/Leaves] 为啥火弦月的8gt刷沙机用不了 (Discussion #321) 什么版本？ — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you authored the thread.Message ID: ***@***.***&gt;

有可能是`leaves.yml`中某个配置导致的
~~说真的 我们不看GitHub Discussions 建议去Discord或者QQ群~~
